tests/testthat/test-DependencyReporter.R

```diff
@@ -28,84 +28,98 @@ test_that('DependencyReporter structure is as expected', {
                , ignore.order = TRUE
                , ignore.case = FALSE
   )
-  
+
   expect_named(object = DependencyReporter$public_fields
                , expected = NULL
                , info = "Available public fields for DependencyReporter not as expected."
                , ignore.order = TRUE
                , ignore.case = FALSE
   )
-  
+
 })
 
 ### USAGE OF PUBLIC AND PRIVATE METHODS AND FIELDS
 
 test_that('DependencyReporter Methods Work', {
   testObj <- DependencyReporter$new()
-  
+
   # inherited set_package
   expect_silent({
-      
+
       # testing set_package with a pkg_path that is relative to the current directory
       entry_wd <- getwd()
-      parent_dir <- dirname(system.file('baseballstats', package='pkgnet'))
+      baseball_dir <- system.file(
+          'baseballstats'
+          , package='pkgnet'
+          , lib.loc = Sys.getenv('PKGNET_TEST_LIB')
+      )
+      parent_dir <- dirname(baseball_dir)
       setwd(parent_dir)
-      
+
       testObj$set_package(
           pkg_name = "baseballstats"
           , pkg_path = 'baseballstats'
       )
       setwd(entry_wd)
   })
-  
+
+  # Sometimes with R CMD CHECK the temp dir begins /private/vars. Other times, just /vars.
+  # Also, sometimes there are double slashes.
+  fmtPath <- function(path){
+      out <- gsub('^/private', '', path)
+      out <- gsub('//', '/', out)
```

Comment:
Is this necessary? Most stuff treat multiple slashes as one slash.

Reply:
Oh nevermind, I see, it's for the test.

Reply:
yeah :/